test/com/m3/octoparts/repository/config/HttpPartConfigRepositorySpec.scala

```diff
@@ -189,6 +190,7 @@ class HttpPartConfigRepositorySpec extends fixture.FunSpec with DBSuite with Mat
         val Some(threadPoolConfig) = hystrixConfig.threadPoolConfig // will fail if not eager loaded
         threadPoolConfig.threadPoolKey should be("swimmingpool")
         threadPoolConfig.coreSize should be(10)
+        threadPoolConfig.queueSize should be(500)
```

Comment:
Good idea. Done as an example of how checking for errors will look like and what the error is.